addons/property_panel/property_panel.gd

```diff
@@ -128,17 +128,18 @@ func _setup_internal_nodes() -> void:
 	@warning_ignore("return_value_discarded")
 	_file_dialog.close_requested.connect(_on_file_dialog_close_requested)
 	add_child(_file_dialog)
+	var style_box := StyleBoxEmpty.new()
+	style_box.set_content_margin_all(5)
```

Comment:
I think this solution is less clean than setting the `offset_X` of the `scroll_container`. Is the result any different?

Reply:
margin:
![image](https://github.com/Jummit/property-panel/assets/52148221/ba730cad-b72d-4f88-93a5-9c7e2daf40cc)
offset:
![image](https://github.com/Jummit/property-panel/assets/52148221/9c5c91be-2f3f-4440-ade6-19ed4498cae7)


> I think this solution is less clean than setting the `offset_X` of the `scroll_container`. Is the result any different?我认为这个解决方案不如设置 `scroll_container` 的 `offset_X` 干净。结果有什么不同吗？

Reply:
This could be a problem caused by the theme or the container nodes which the panel is inside in. Could you share the project or a scene file so I can take a look? I think it'll be easier to figure out why this is happening.

Reply:
> This could be a problem caused by the theme or by the container nodes which the panel is inside in. Could you share the project or a scene file so I can take a look? I think it'll be easier to figure out why this is happening, as I can't reproduce this issue myself.这可能是由主题或面板所在的容器节点引起的问题。您可以共享项目或场景文件，以便我看一下吗？我认为弄清楚为什么会这样会更容易，因为我自己无法重现这个问题。

test.tscn
![image](https://github.com/Jummit/property-panel/assets/52148221/684c8bdc-e1f1-46c1-8f77-6e3edefd8a4b)
![image](https://github.com/Jummit/property-panel/assets/52148221/6b25f505-8244-4810-850d-34161388e6ba)

Reply:
Thanks, I see what you mean. I think the best solution would be to put the `ScrollContainer` into a `MarginContainer`, and then remove the `scroll_container.offset_` code.